main/equations.go

```diff
@@ -288,10 +288,24 @@ More accurate over longer distances
 // distance calculates distance between two points
 // using law of cosines
 
-func distance(lat1, lon1, lat2, lon2 float64) (float64, bool) {
-	// radius_earth := 6371008.8 // meters; mean radius
+func distance(lat1, lon1, lat2, lon2 float64) (dist, bearing float64) {
```

Comment:
https://github.com/cyoung/stratux/blob/master/test/nexrad_annunciator.go
https://godoc.org/github.com/kellydunn/golang-geo

This package has a pretty nice interface and very useful methods, `BearingTo()`, `GreatCircleDistance()`, and can draw polygon sets and test set membership of points.